src/app/(drift)/(auth)/components/index.tsx

```diff
@@ -127,28 +128,29 @@ function Auth({
 						<Button width={"100%"} type="submit" loading={submitting}>
 							Sign {signText}
 						</Button>
-						{isGithubEnabled ? (
-							<>
-								<hr style={{ width: "100%" }} />
-								<Button
-									type="submit"
-									width="100%"
-									style={{
-										color: "var(--fg)"
-									}}
-									iconLeft={<GitHub />}
-									onClick={(e) => {
-										e.preventDefault()
-										signIn("github", {
-											callbackUrl: "/",
-											registration_password: serverPassword
-										})
-									}}
-								>
-									Sign {signText.toLowerCase()} with GitHub
-								</Button>
-							</>
-						) : null}
+						<>
+							{authProviders?.map((provider) => {
```

Comment:
This will render `undefined` if `authProviders` is empty or undefined.

Can you change it to:

`{autoProviders?.length ? <> {authProviders.map...} : null}`?

Reply:
good point, thank you